_posts/blog/2019-06-20-Hello!I am Yini He.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Hello^0^I am Yini He!
+categories: blog
+excerpt: An introduction to the Whitaker lab's newest member
+image:
+  feature:
+link:
+date: 2019-06-20
+modified:
+share: true
+author: Yini-He
+---
+
+## Who am I
+
+--I am who I am（我就是我）
+
+My name is Yini He（何旖旎）.I’m a PhD student with a major in neuroimaging at the University of Electronic Science and Technology of China (UESTC).It is truly amazing for me to study in the WhitakerLab at the Alan Turing Institute as a visiting student,jointly funded by Chinese Scholarship Council (CSC) and the Newton Fund.I am enthusiastic about  the relationship between individual traits, the brain and emotions, and particularly the associated risks of developing mental disorders.
+
+
+## Where I come here  
+
+--As different as night and day（冰火两重天）
+
+I was born in Changsha city of China which famous for the spicy dishes and excessively distinct seasons. And then I moved to Haikou, which famous for the gentlest sea wind and spring-like seasons. As its title indicates, these two cities have extreme difference, just like a song plays from a gentle and poetic melody to passionate and fiery bursts, fully influencing the two sides of my personality. I am happy and passionate at work, at the same time, enjoy a totally peaceful and quiet life. 
+
+<figure>
+  <img src="/images/YiniHe/1coconut4.jpg">
```

Comment:
Can you update the image to include the link to image on the internet? You can also use "<figcaption>" to caption the image, it doesn't need to be in the picture itself.